src/Lean/Server/InfoUtils.lean

```diff
@@ -254,7 +255,7 @@ partial def InfoTree.hoverableInfoAt? (t : InfoTree) (hoverPos : String.Pos) (in
     -/
     if info.stx.isOfKind nullKind || info.toElabInfo?.any (·.elaborator == `Lean.Elab.Tactic.evalWithAnnotateState) then
       return results
-    unless (info matches .ofFieldInfo _ | .ofOptionInfo _ || info.toElabInfo?.isSome) && info.contains hoverPos includeStop do
+    unless (info matches .ofFieldInfo _ | .ofOptionInfo _ | .ofErrorNameInfo _ || info.toElabInfo?.isSome) && info.contains hoverPos includeStop do
```

Comment:
This isn't a huge deal, but I think this prevents a term goal from being displayed for terms tagged with `ErrorNameInfo`, since `hoverableInfoAt?` will select the `ErrorNameInfo` over the `TermInfo` ? I suppose one could add another flag to the function ...

Reply:
(Honestly, I don't think it's worth investing time to fix this, just something I noticed)

Reply:
I don't think this affects the existing macros, though, right? I still see term goals using `throwNamedError` and `throwNamedErrorAt` once the input is sufficient for the non-error-name parts to be parsed as terms.